app/controllers/admin/questions_controller.rb

```diff
@@ -53,13 +54,13 @@ def update
   def destroy
     Question.find(params[:id]).destroy
     flash[:success] = 'Question deleted'
-    redirect_to admin_questions_path
+    redirect_to :back rescue redirect_to admin_questions_path
```

Comment:
We need :back here because when we delete question from table in topic we don't need to be redirected to all questions. But if i leave here only :back  test would't pass.